### PR TITLE
[MIG] product_pricelist_by_contact: migration to 16

### PR DESCRIPTION
**Pull request by martiqubiq**

### CODE REVIEW

**Review by hugo-cordoba — APPROVED** [review]
LGTM!

**Comment by valjaro** [comment]
LGTM

File: product_pricelist_by_contact/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Pricelist Per Contact",
+    "version": "16.0.0.0.0",
```
Comment (pedrobaeza) [inline product_pricelist_by_contact/__manifest__.py:6]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by pedrobaeza — COMMENTED** [review]
/ocabot migration product_pricelist_by_contact

File: product_pricelist_by_contact/__manifest__.py (hunk 1 above)
Comment (hugo-cordoba) [reply product_pricelist_by_contact/__manifest__.py:6]: @martiqubiq could you change this pls?

**Comment by pedrobaeza** [comment]
ping @martiqubiq

**Comment by cubells** [comment]
Supersed by https://github.com/OCA/product-attribute/pull/2052

**Pull request closed without merging**